lib/scala/logging-service/src/main/scala/org/enso/loggingservice/internal/DefaultLogMessageRenderer.scala

```diff
@@ -70,5 +70,6 @@ class DefaultLogMessageRenderer(printExceptions: Boolean)
       case LogLevel.Debug   => "debug"
       case LogLevel.Trace   => "trace"
       case LogLevel.Off     => "off"
+      case _                => "error";
```

Comment:
I don't think that's necessary since the pattern match here is exhaustive

Reply:
Exhaustive, except not covering `null`.